Installing extension composer or legacy (#5305)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ExtensionArchitecture/HowTo/CreateNewExtension.rst

Starting with TYPO3 v11 it is no longer possible to install extensions in TYPO3
without using Composer in Composer-based installations.

However during development it is necessary to test your extension locally
before publishing it. Place the extension directory in a directory called,

> :file:`packages` in TYPO3's root directory. You can freely name the extension directory.

Reviewer: I know, it's the text of the original, but what about `TYPO3 project`? It makes it more clear, that the directory should not be created somewhere in `vendor/typo3/` or `public/typo3/`

```suggestion
:file:`packages` in TYPO3 projects root directory. You can freely name the extension directory.
```

..  code-block:: bash

    composer req my-vendor/my-extension:"@dev"

..  hint::
    Starting with TYPO3 v11.5 all extensions installed via Composer are
    automatically activated when they are installed.

..  hint::

>     For legacy installations you can put the extension directly in the directory

Reviewer 2: ```suggestion
    For Classic installations you can put the extension directly in the directory
```

    :file:`typo3conf/ext` and then **activate** it in

>     :guilabel:`Admin Tools > Extension Manager`.

Reviewer: I go back until TYPO3 9.5. The modules name is "Extensions":

```suggestion
    :guilabel:`Admin Tools > Extensions`.
```
